app/services/Metrics.scala

```diff
@@ -2,30 +2,27 @@ package services
 
 import metrics.{CloudWatch, CloudWatchMetrics}
 import play.api.inject.ApplicationLifecycle
-import play.api.{Environment, Mode}
-import rx.lang.scala.Observable
+import rx.lang.scala.{Observable, Subscription}
 
 import scala.concurrent.Future
 import scala.concurrent.duration._
 
-class Metrics(cloudWatch: CloudWatch, stage: String, agents: Agents, lifecycle: ApplicationLifecycle) {
+class Metrics(cloudWatch: CloudWatch, namespace: String, securityHqNamespace: String, agents: Agents, lifecycle: ApplicationLifecycle) {
+  val subscription: Subscription = Observable.interval(initialDelay = 10.seconds, period = 6.hours).subscribe { _ =>
+    cloudWatch.put(namespace, CloudWatchMetrics.OldCount.name, agents.oldProdInstanceCount)
+    cloudWatch.put(namespace, CloudWatchMetrics.AmisAgePercentile25th.name, agents.amisAgePercentiles.flatMap(_.p25))
+    cloudWatch.put(namespace, CloudWatchMetrics.AmisAgePercentile50th.name, agents.amisAgePercentiles.flatMap(_.p50))
+    cloudWatch.put(namespace, CloudWatchMetrics.AmisAgePercentile75th.name, agents.amisAgePercentiles.flatMap(_.p75))
+    cloudWatch.put(namespace, CloudWatchMetrics.AmisAgePercentile90th.name, agents.amisAgePercentiles.flatMap(_.p90))
+    cloudWatch.put(namespace, CloudWatchMetrics.AmisAgePercentileHighest.name, agents.amisAgePercentiles.flatMap(_.highest))
 
-  // only add metrics from PROD
-  if (stage == "PROD") {
-
-    val subscription = Observable.interval(initialDelay = 10.seconds, period = 6.hours).subscribe { _ =>
-      cloudWatch.put(CloudWatchMetrics.OldCount.name, agents.oldProdInstanceCount)
-      cloudWatch.put(CloudWatchMetrics.AmisAgePercentile25th.name, agents.amisAgePercentiles.flatMap(_.p25))
-      cloudWatch.put(CloudWatchMetrics.AmisAgePercentile50th.name, agents.amisAgePercentiles.flatMap(_.p50))
-      cloudWatch.put(CloudWatchMetrics.AmisAgePercentile75th.name, agents.amisAgePercentiles.flatMap(_.p75))
-      cloudWatch.put(CloudWatchMetrics.AmisAgePercentile90th.name, agents.amisAgePercentiles.flatMap(_.p90))
-      cloudWatch.put(CloudWatchMetrics.AmisAgePercentileHighest.name, agents.amisAgePercentiles.flatMap(_.highest))
-      cloudWatch.put(CloudWatchMetrics.OldCountByAccount.name, agents.oldInstanceCountByAccount)
-    }
+    // These metrics are used by a Grafana dashboard which graphs SecurityHQ related metrics
+    // TODO use the same namespace as other metrics, and filter in Grafana instead?
+    cloudWatch.put(securityHqNamespace, CloudWatchMetrics.OldCountByAccount.name, agents.oldInstanceCountByAccount)
+  }
```

Comment:
Every stage will produce the same metric on the same namespace. That is there will be duplication. Should we only produce metrics on `securityHqNamespace` in PROD?